### PR TITLE
Helm Chart for lisk-core - Closes #2986

### LINKED ISSUE #2986: Helm chart for kubernetes deployment
### Expected behavior
Lisk core should be able to be installed on kubernetes

### Actual behavior
Theres is no kubernetes installation option yet

### Steps to reproduce
N/A

### Which version(s) does this affect? (Environment, OS, etc...)
N/A

### PR DESCRIPTION
**Pull request by psychopenguin**
### What was the problem?

New feature. Option to install lisk-core on kubernetes using helm. See #2986 

### How did I fix it?

Creating a helm chart

### How to test it?

Deploy the chart on a kubernetes cluster.

### Review checklist

* The PR resolves #2986 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: helm/lisk-core/values.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  wsPort: 7001
+  httpPort: 7000
+  api:
+    public: true
+  database:
+    user: lisk
+    password: lisk
+  image:
+    repository: lisk/core
+    tag: 1.5.0
```
Comment (fchavant) [inline helm/lisk-core/values.yaml]:
```suggestion
    tag: 1.6.0-alpha.5
```

File: helm/lisk-core/values.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+replicaCount: 1
+
+persistence:
+  enable: true
+  storage: 10Gi
```
Comment (fchavant) [inline helm/lisk-core/values.yaml]: We should add a comment explaining this is used for postgrtes and can be expanded.

File: helm/lisk-core/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+Create a yaml file with the variables you want to change - use `values.yaml` as template.
+
+If you want to connect to mainnet for example:
+
+```
+lisk:
+    network: mainnet
+    wsPort: 8001
+    httpPort: 8000
+```
```
Comment (fchavant) [inline helm/lisk-core/README.md]: Mainnet needs more disk space for its database.

File: helm/lisk-core/README.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+- A kubernetes cluster with helm tiller deployed
+- Support for persistent storage
+
+## Installation
+
+Inside chart main directory - where `Chart.yaml` is located:
+
+### Install dependencies
+
+`helm dependency build`
```
Comment (fchavant) [inline helm/lisk-core/README.md]: Is this really necessary?
```
~> helm dependency build
No requirements found in ./charts.
```

Comment (psychopenguin) [reply helm/lisk-core/README.md]: my bad. it was necessary in the initial version I did, when postgresql was running in a different pod from lisk core. will remove it.

**Comment by psychopenguin** [comment]
[Plan for lisk-service in production](https://trello.com/c/yIdEVz0F/346-plan-for-lisk-service-in-production)

**Pull request merged**